<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="3" hostname="sachintaw-w10" name="com.cybage.assignment.test.widgetsPageTest" tests="7" failures="0" timestamp="06 Oct 2021 09:12:58 GMT" time="33.746" errors="1">
  <testcase name="validatePresenceOfWidgetOptions" time="3.744" classname="com.cybage.assignment.test.widgetsPageTest"/>
  <testcase name="validateAccordianPageOperations" time="3.072" classname="com.cybage.assignment.test.widgetsPageTest">
    <skipped/>
  </testcase> <!-- validateAccordianPageOperations -->
  <testcase name="validateDatePickerOperation" time="9.166" classname="com.cybage.assignment.test.widgetsPageTest"/>
  <testcase name="validateAutoCompleteTextBoxes" time="6.926" classname="com.cybage.assignment.test.widgetsPageTest"/>
  <testcase name="validateAccordianPageOperations" time="3.011" classname="com.cybage.assignment.test.widgetsPageTest">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.cybage.assignment.objects.utilities.readWorkbook(utilities.java:70)
at com.cybage.assignment.test.widgetsPageTest.validateAccordianPageOperations(widgetsPageTest.java:68)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.retryFailed(Invoker.java:839)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1010)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </error>
  </testcase> <!-- validateAccordianPageOperations -->
  <testcase name="validateAccordianPageOperations" time="4.801" classname="com.cybage.assignment.test.widgetsPageTest">
    <skipped/>
  </testcase> <!-- validateAccordianPageOperations -->
  <testcase name="validateAccordianPageOperations" time="3.026" classname="com.cybage.assignment.test.widgetsPageTest">
    <skipped/>
  </testcase> <!-- validateAccordianPageOperations -->
</testsuite> <!-- com.cybage.assignment.test.widgetsPageTest -->
